### ISSUE TITLE
Crash when mouse controls are accessed from keyboard window

### ISSUE THREAD

**Issue by rtiainen** [dirkwhoffmann/vAmiga#550]
When a keyboard window is opened and mouse controls on upper right control are accessed the vAmiga crashes. Steps to reproduce:

 - Open vAmiga 1.1b3
 - Power on the emulation
 - Open keyboard window from tooltip or Keyboard -> Show menu
 - Click either mouse icon on the right upper corner
 - Observe the crash

Here is the crash report:

```
Process:               vAmiga [75751]
Path:                  /private/var/folders/*/vAmiga.app/Contents/MacOS/vAmiga
Identifier:            dirkwhoffmann.vAmiga
Version:               1.1b3 (1.1b3)
Code Type:             X86-64 (Native)
Parent Process:        ??? [1]
Responsible:           vAmiga [75751]
User ID:               501

Date/Time:             2021-09-28 23:15:29.812 +0300
OS Version:            macOS 11.6 (20G165)
Report Version:        12
Anonymous UUID:        E682E6AC-6E14-9912-D049-29375D5DE1DF


Time Awake Since Boot: 630000 seconds

System Integrity Protection: enabled

Notes:                 Translocated Process

Crashed Thread:        0  Dispatch queue: com.apple.main-thread

Exception Type:        EXC_CRASH (SIGABRT)
Exception Codes:       0x0000000000000000, 0x0000000000000000
Exception Note:        EXC_CORPSE_NOTIFY

Application Specific Information:
Assertion failed: (keycode < 0x80), function pressKey, file /Users/hoff/vAmiga/Emulator/Peripherals/Keyboard/Keyboard.cpp, line 118.
```

**Comment by dirkwhoffmann** [comment]
Confirmed. When one of those images is clicked, the tag of the corresponding GUI element is send to the emulator as a key code. This is obviously wrong, because clicking the images should not have any effect.

**Comment by dirkwhoffmann** [comment]
Will be fixed in V1.1.

**Comment by dirkwhoffmann** [comment]
Fixed in vAmiga 1.1b4
